app/models/shipit/deploy_spec/bundler_discovery.rb

```diff
@@ -35,7 +35,7 @@ def remove_ruby_version_from_gemfile
           # OSX is nitpicky about the -i.
           %q(/usr/bin/sed -i '' '/^ruby\s/d' Gemfile)
         else
-          %q(sed -i '/^ruby\s/d' Gemfile)
+          %q(sed -i '/^ruby(\s|\s\(|\()/d' Gemfile)
```

Comment:
```suggestion
          %q(sed -i '/^ruby(\s|\()/d' Gemfile)
```

https://shopify.github.io/ruby-style-guide/
`I Do not put a space between a method name and the opening parenthesis.`

Reply:
Ah. I wasn't quite sure. Makes this a bit simpler :)